pkg/tests/apis/provisioning/provisioning_test.go

```diff
@@ -281,14 +282,18 @@ func TestIntegrationProvisioning_ImportAllPanelsFromLocalRepository(t *testing.T
 	// Now, we import it, such that it may exist
 	helper.SyncAndWait(t, repo, nil)
 
-	found, err := helper.Dashboards.Resource.List(ctx, metav1.ListOptions{})
-	require.NoError(t, err, "can list values")
+	require.EventuallyWithT(t, func(collect *assert.CollectT) {
```

Comment:
was this an effort to avoid the flakey test?

if 283 passes, we expect the read-after-write to support List, so I don't think it should be within an eventually clause

Reply:
Yes exactly. Shall I revert this, considering you've submitted your fix?

Reply:
I've reverted them.